src/main/resources/locales/en/translation/accessibility-statement.json

```diff
@@ -0,0 +1,72 @@
+  "paragraph_s9_1": "The content listed below is non-accessible for the following reasons.",
+  "nonComplianceReason1": "Non-compliance with the accessibility regulations",
+  "nonCompliance_1_1": "On some pages the focus positioning is not on the expected areas which fail to meet WCAG 2.1A success criterion 2.4.3 Focus Order Level A.",
+  "nonCompliance_1_2": "On some pages, links did not have clear descriptions of their purpose and destinations e.g. lack of information on whether the link opens in a new tab which fail to meet WCAG 2.1A success criterion 2.4.4 Link Purpose Level A.",
+  "nonCompliance_1_3": "Some fields do not have associated labels which affects screen readers and voice activation users. This fails to meet WCAG 2.1A success criteria 1.3.1 Info and Relationships Level A, 4.1.2 Name, Role, Value Level A and 2.5.3 Label in Name – Level A.",
+  "nonCompliance_1_4": "On some pages, the heading levels are not in the correct order which fail to meet WCAG 2.1A success criterion 1.3.1 Info and Relationships Level A.",
+  "nonCompliance_1_5": "Some items navigated to via the tab key on the keyboard do not have very distinct highlighting which is sometimes too hard for low vision users to determine. This fails to meet WCAG 2.1A success criterion 1.4.11 Non-text Contrast Level AA.",
+  "nonCompliance_1_6": "Screen reader users found radio buttons lacked fieldset and legend to provide the accompanying question when navigating in and out of context. This fails 1.3.1 Info and Relationships – Level A and 2.4.6 Headings and Labels – Level AA",
+  "paragraph_s9_2": "We plan to rectify the issues listed above by March 2023.",
+  "nonComplianceReason2": "Content that’s not within the scope of the accessibility regulations",
+  "nonComplianceReason2_PDF_DOCS": "PDFs and other documents",
+  "paragraph_s9_3": "Some of our PDFs and Word documents are essential to providing our services. For example, we have PDFs with information on how users can access our services, and forms published as PDF/Word documents.",
+  "paragraph_s9_4": "The accessibility regulations <a href=\"https://www.legislation.gov.uk/uksi/2018/952/regulation/4/made\">do not require us to fix PDFs or other documents published before 23 September 2018</a> if they are not essential to providing our services.",
+  "paragraph_s9_5": "Any new PDFs or Word documents we publish will meet accessibility standards.",
+  "header10": "What we’re doing to improve accessibility",
+  "paragraph_s10_1": "This website is currently being improved based on the digital accessibility report. We will fix the accessibility issues identified in the report and update this statement regularly. The site is continually tested using accessibility software internally and externally. Any new features which are introduced will also be tested.",
+  "header11": "Preparation of this accessibility statement",
+  "paragraph_s11_1": "This statement was prepared on [date when it was first published].",
```

Comment:
do we need an actual date for [date when it was first published]

Reply:
It is written like that in the [HTML file](https://tools.hmcts.net/jira/secure/attachment/456039/ET%20accessibility%20statement.html) that we received from business (George Mitchell). I don' t think that it mentions the actual date but I think we need to write something 18/01/2023